src/components/BulkEnrollmentPage/stepper/BulkEnrollmentSubmit.jsx

```diff
@@ -24,8 +24,6 @@ import { ToastsContext } from '../../Toasts';
 import { clearSelectionAction } from '../data/actions';
 import { configuration } from '../../../config';
 
-export const BULK_ENROLL_ERROR = 'There was an ';
```

Comment:
was this just unused already?

Reply:
yeah, I couldn't find any usages for it, not sure how it got there. You should be able to see on master that it's just there unused.